java/ql/lib/semmle/code/java/controlflow/Guards.qll

```diff
@@ -218,12 +218,12 @@ predicate guardControls_v1(Guard guard, BasicBlock controlled, boolean branch) {
 }
 
 /**
- * INTERNAL: Use `Guards.controls` instead.
+ * DEPRECATED: Use `Guards.controls` instead.
  *
  * Holds if `guard.controls(controlled, branch)`, except this doesn't rely on
  * RangeAnalysis.
  */
-predicate guardControls_v2(Guard guard, BasicBlock controlled, boolean branch) {
+deprecated predicate guardControls_v2(Guard guard, BasicBlock controlled, boolean branch) {
```

Comment:
I'd like to keep this for completeness, I think. Sure it's not used, but that's ok.